test/test_KubeLibrary.py

```diff
@@ -247,47 +248,178 @@ def mock_list_namespaced_role_bindings(namespace, watch=False):
 
 k8s_api_url = 'https://0.0.0.0:38041'
 
+responses = Responses('requests.packages.urllib3')
+
 
 class TestKubeLibrary(unittest.TestCase):
 
     apis = ('v1', 'extensionsv1beta1', 'batchv1', 'appsv1', 'batchv1_beta1',
```

Comment:
and then dynamic client would land here as another thing to test

Reply:
Agreed.